lib/zendesk_apps_tools/command.rb

```diff
@@ -85,7 +89,7 @@ def validate
 
       if valid
         app_package.warnings.each { |w| say w.to_s, :yellow }
-        deprecated_message
+        deprecated_message unless options[:'no-deprecate']
```

Comment:
Can you rename this method, it seems like it would only log a message. Also, how about `bypass-deprecation`?

Reply:
Also, maybe add a comment and create a JIRA to remove this once all official apps are migrated to v2?

Reply:
> Can you rename this method, it seems like it would only log a message

Actually i was wrong when you asked, it does either log or log and exit depending on the message.

Reply:
Still don't like that it's named `deprecated_message`, perhaps `deprecation_check`?